common/lib/dependabot/errors.rb

```diff
@@ -243,6 +243,11 @@ def self.updater_error_details(error)
         "error-type": "git_token_auth_error",
         "error-detail": { message: error.message }
       }
+    when URI::InvalidURIError
+      {
```

Comment:
should this be a `MisconfiguredTooling` error instead since the yml file contains incorrect url settings.

Reply:
Or maybe `DependencyFileNotResolvable`; see this [PR](https://github.com/dependabot/dependabot-core/commit/84d105e61c4233fdcc91d133b7774135c486b972)

Reply:
@abdulapopoola , valid proposition , ill change it to DependencyFileNotResolvable